### ISSUE TITLE
Mycosphaeroides is extinct ...

### ISSUE THREAD

**Issue by hyanwong** [OpenTreeOfLife/feedback#54]
... but appears in the synthetic tree
# 

| Metadata | Do not edit below this line |
| :-- | :-- |
| Author | [hyanwong](https://github.com/hyanwong) |
| Upvotes | 0 |
| URL | [tree.opentreeoflife.org/opentree/otol.draft.22@846805/Mycosphaeroides](http://tree.opentreeoflife.org/opentree/otol.draft.22@846805/Mycosphaeroides) |
| Target node label | Mycosphaeroides |
| Synthetic tree id | otol.draft.22 |
| Synthetic tree node id | 846805 |
| Source tree id(s) | if:585156,gbif:4885184,irmng:1089658 |
| Open Tree Taxonomy id | 4115953 |
| Intended scope | Re: OTT taxon |

**Comment by jar398** [comment]
Thanks. This isn't marked extinct in the version of IRMNG (Jan 31) that we
used to get extinctness annotations, but IRMNG does now seem to know it's
extinct. The best way to get this fixed is to obtain a recent IRMNG dump.
I'll see if I can do that.​

Index fungorum has
_Position in classification:_
Fossil Mycetozoa

Probably this information was not in the database dump that we received.

**Comment by jar398** [comment]
Tracking this (and the other two) as
https://github.com/OpenTreeOfLife/reference-taxonomy/issues/116

**Comment by hyanwong** [comment]
On 16 Dec 2014, at 15:56, Jonathan A Rees notifications@github.com wrote:

> Tracking this (and the other two) as 
> https://github.com/OpenTreeOfLife/reference-taxonomy/issues/116 

Thanks. In fact, one of the main problems as I got through the tree seems to be a smattering of extinct species which clutter up an otherwise conventional topology. Is there a good way to mark these up: e.g. provide a checkbox on the feedback form, or perhaps add them to a list on the github issue/116 page. What would suit you best?=

**Comment by jar398** [comment]
When I get back to working on the taxonomy I will go through all the
reference-taxonomy issues. So adding to issue 116 is good, or creating a
new issue there. A single list would be best since I can just transform it
into the form required by the patch system. Or if you want to learn the
patch system (
https://github.com/OpenTreeOfLife/reference-taxonomy/blob/master/doc/scripting.md).
..

I will probably go through the feedback issues as well, but may forget to.

I checked to see if there's a new IRMNG release and there isn't yet. If I
can convince Tony (or whoever is in charge now) to make one we may be able
to fix large numbers of these in one fell swoop, without you having to find
them manually. You might check the IRMNG web site to see what it now knows
about these taxa. The ones I checked were already marked extinct.

Jonathan
